dev/Sumangrewal/main.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import confusion_matrix,accuracy_score
+
+dataset = pd.read_csv("winequality.csv")
+
+
+
+x=dataset.iloc[:,:-2].values
+y=dataset.iloc[:,12].values
+
+x_train,x_test,y_train,y_test=train_test_split(x,y,test_size=0.25)
```

Comment:
Please add a comment (in the notebook) explaining your choice of this test size.

Reply:
Changes made in these too.